shopping_mall/backend/tests/conftest.py

```
@@ -63,6 +63,18 @@ def retrieve_multiple(
                     docs.append(doc)
         return docs
 
+    def retrieve_with_metadata(
+        self,
+        question: str,
```

Ruff (0.15.12)

[warning] Unused method argument: `question`

(ARG002)